backend/src/models/MedicationCategory.js

```diff
@@ -0,0 +1,32 @@
+import mongoose from 'mongoose';
+import {
+  PILLS_COLOR,
+  SYRUP_COLOR,
+  INHALER_COLOR,
+  INJECTION_COLOR,
+  DROPS_COLOR,
+  PATCHES_COLOR,
+} from '../constants/pillColors';
+
+import { PILLS_ICON, SYRUP_ICON, INHALER_ICON, INJECTION_ICON, DROPS_ICON, PATCHES_ICON } from '../constants/medsIcons';
+
+const { Schema } = mongoose;
+
+const MedicationCategorySchema = new Schema({
```

Comment:
Keep the convention that schema variable is camelcase (medicationCategorySchema)